packages/webpack/README.md

```diff
@@ -11,7 +11,9 @@ LavaMoat Webpack Plugin wraps each module in the bundle in a [Compartment](https
 1. Generate a policy file for your app's entry `npx lavamoat --autopolicy path/to/index.js` and adjust it if needed.
 2. Note that `builtins` in the policy indicate modules that webpack will either polyfill or ignore, so they need to be moved to `packages` and explicitly added as dependencies or just removed.
 3. Create a webpack bundle with the LavaMoat plugin enabled and pass the policy from the policy file to it in options.
-4. Make sure you add a `<script src="./lockdown.js"></script>` before all other scripts or enable the `HtmlWebpackPluginInterop` option if you're using `html-webpack-plugin`.
+4. Make sure you add a `<script src="./lockdown"></script>` before all other scripts or enable the `HtmlWebpackPluginInterop` option if you're using `html-webpack-plugin`.
+
+> The pligin is emitting lockdown without the `.js` extension because that's the only way to prevent it from getting minified, which is likely to break it.
```

Comment:
```suggestion
> The plugin is emitting lockdown without the `.js` extension because that's the only way to prevent it from getting minified, which is likely to break it.
```